operator-common/src/main/java/io/strimzi/operator/common/Annotations.java

```diff
@@ -80,6 +80,17 @@ public class Annotations extends ResourceAnnotations {
      */
     public static final String ANNO_DEP_KUBE_IO_REVISION = "deployment.kubernetes.io/revision";
 
+    /**
+     * Annotation for connector offsets
+     */
+    public static final String ANNO_STRIMZI_IO_CONNECTOR_OFFSETS = STRIMZI_DOMAIN + "connector-offsets";
+
+    /**
+     * Annotation for identifying MirrorMaker2 connector when managing connector offsets
+     */
+    public static final String ANNO_STRIMZI_IO_MIRRORMAKER_CONNECTOR = STRIMZI_DOMAIN + "mirrormaker-connector";
+
```

Comment:
Should this be in `ResourceAnnotations`?

Reply:
Good point, moved